doc/developer-guides/hld/hv-dev-passthrough.rst

```diff
@@ -102,12 +124,32 @@ following figures:
 
    ptdev de-assignment control flow
 
+VT-d Interrupt-remapping
+************************
+
+The VT-d interrupt-remapping architecture enables system software to
+control and censor external interrupt requests generated by all sources
```

Comment:
generated by -> from?

Reply:
This sentence is from VT-d spec.  Will leave it as it is.